### ISSUE TITLE
WebView2 have a border

### ISSUE THREAD

**Issue by X-Yunner** [reactivemarbles/CrissCross#135]
I have two Windows: loginwindow and mainwindow. webview2 In the mainwindow, I first display the loginwindow and it will have a border. 
![Snipaste_2024-07-24_14-07-37](https://github.com/user-attachments/assets/61532628-7921-48a3-8584-8ff47b4bd46d)

After the loginwindow disappears, there will still be a border in the mainwindow. How can I cancel this border?
![Snipaste_2024-07-24_14-16-01](https://github.com/user-attachments/assets/cce93b8c-cabf-4e61-a830-f45eec9e6c53)

![Snipaste_2024-07-24_14-11-56](https://github.com/user-attachments/assets/80c25dbd-3026-4a29-814b-afde58897b7a)

code
![image](https://github.com/user-attachments/assets/c41859c2-69df-4185-96b4-e672af08ddd8)

**Comment by ChrisPulman** [comment]
Thank you for raising this, I will investigate what might be the cause of the border and provide and update if there is a way to resolve this issue.

**Comment by ChrisPulman** [comment]
@X-Yunner Please could you try V2.0.2 and see if the issue still exists, if it doesn't, would it be possible to put a minimal demonstration of the issue to assist me to see where the issue is coming from.

**Comment by X-Yunner** [comment]
Thank you for handling the problem so quickly. I created a demo library in my github repository, which can reproduce this problem. 
[](https://github.com/X-Yunner/TestWebView2)
I checked for distribution problems myself. It seems that the wpf-ui library appears. If I use this ui library, a border will appear in webview2. Maybe I will go to the wpf-ui library and ask questions. Anyway, thanks.

**Comment by ChrisPulman** [comment]
> Thank you for handling the problem so quickly. I created a demo library in my github repository, which can reproduce this problem. [X-Yunner/TestWebView2](https://github.com/X-Yunner/TestWebView2) I checked for distribution problems myself. It seems that the wpf-ui library appears. If I use this ui library, a border will appear in webview2. Maybe I will go to the wpf-ui library and ask questions. Anyway, thanks.

@X-Yunner I created a PR in your repository, hopefully we can find what is happening in your app.

**Comment by X-Yunner** [comment]
Yes, I have seen Pr, it seems that the different versions of the computer system caused by the border problem

**Comment by ChrisPulman** [comment]
Which OS are you using?

**Comment by X-Yunner** [comment]
Looking forward to a solution

**Comment by ChrisPulman** [comment]
@X-Yunner please update to V2.0.3 of CrissCross, I believe I found the issue you were experiencing with Windows 10, I have made the functionality equal for Windows 10 or 11.

You may wish to consider using a MVVM approach and have a single Wpf Window per physical monitor as a Navigation Host window for displaying a multitude of UserControls (Views) controlled with navigation.

**Comment by X-Yunner** [comment]
Yes, thank you for fixing it. There is no border in the loginwindow interface, but there is still a border in the mainwindow. Is there a way to cancel it?
![Snipaste_2024-07-29_09-27-50](https://github.com/user-attachments/assets/32aa6eeb-8bbc-4f10-a494-2dd62915439d)

**Comment by ChrisPulman** [comment]
@X-Yunner I will try to see where that comes from, it looks like a scroll viewer which I do not add, but if I can locate the source and control it, I will let you know how I get on.

**Comment by ChrisPulman** [comment]
@X-Yunner I believe that the base WebView2 component has been updated to include this scroll component and is there as a part of the Microsoft control which has no exposure to the external API to control it. The border you are seeing is part of the scroll viewer, the UX appears differently between Windows 10 and Windows 11 as you have experienced. Hopefully your client can update to Windows 11 and solve the issue for us :)
I have raised the issue within the Microsoft Webview2 control, but unfortunately nothing happens quickly there.

**Comment by X-Yunner** [comment]
Is this the problem you are talking about? If I don't include a control in it, it will have a True appear.

![image](https://github.com/user-attachments/assets/2943404d-dccd-4163-8bdf-2e1d8d30f355)

**Comment by ChrisPulman** [comment]
@X-Yunner I will see if I can find the true, that is definitely a strange one. I will focus on this tonight

**Comment by ChrisPulman** [comment]
Closing as found to be a core WebView 2 issue
